subxt/src/events/mod.rs

```diff
@@ -14,7 +14,63 @@
 // You should have received a copy of the GNU General Public License
 // along with subxt.  If not, see <http://www.gnu.org/licenses/>.
 
-//! For working with events.
+//! Parse the events generated by a block.
+//!
+//! This module exposes the ability to work with events generated by a given block.
+//! There are two types of events that the subxt is internally aware of:
+//! - static events: Events that are already present in the subxt's client metadata
+//! - dynamic events: Events that are present only in the target node's metadata
```

Comment:
This feels a little misleading; they aren't different types of events, just different wayt of working with the events that come back. Maybe something more like:

```
```suggestion
//!
//! Subxt can either attempt to statically decode events into known types, or it
//! can hand back details of the raw event without knowing what shape its contents 
//! are (this may be useful if we don't know what exactly we're looking for).
```